src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -323,4 +325,12 @@ private static String resolvePostCode(SscsCaseData sscsCaseData) {
 
         return sscsCaseData.getAppeal().getAppellant().getAddress().getPostcode();
     }
+
+    private void updateHearingOptionLanguageFromSelectedList(SscsCaseData sscsCaseData) {
+        HearingOptions hearingOptions = sscsCaseData.getAppeal().getHearingOptions();
+        if (hearingOptions != null) {
+            DynamicList languagesList = hearingOptions.getLanguagesList();
+            hearingOptions.setLanguages(isNull(languagesList) ? "" : languagesList.getValue().getLabel());
```

Comment:
Is passing an empty string desirable here? What needs to happen for the list to be null here? Is there not a default value we can fall back on instead?

Reply:
This is very similar to what's done during the adjournCase event. The original idea was to convert the text field to a drop-down field, but the work required way too many changes. So we decided to keep the original text field (but hide it from the user) and just set it to whatever value was chosen from the dropdown list. If a new case is created in the future, both the text field and dropdown will be empty or null. Besides, the language text field was optional, so setting it to an empty string should be fine.

Reply:
Cool, good explanation, thanks.